First batch of adding descriptions to the documentation (#14825)
netdata/netdata: docs/cloud/visualize/interact-new-charts.md

# Interact with charts

> Interacting with the charts will help you make sense of all the metrics at your disposal, helping you troubleshoot with real-time, per-second metric data.

Reviewer: ```suggestion
Learn how to use Netdata's powerful charts to troubleshoot with real-time, per-second metric data.
```
